.github/workflows/assign.yml

```diff
@@ -0,0 +1,15 @@
+name: Assign
+on:
+  issue_comment:
+    types: created
+
+jobs:
+  one:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+            if [[ "${{ github.event.comment.body }}" == "take" ]]; then
```

Comment:
how about ``take-issue`` or ``assign-issue`` (then in theory could actually specify a username after)

Reply:
For the keyword, I found `take` the simplest option (thought of assign too, but I think it's easier to make typos, asign, assing...).

I thought about adding a user after it, but we (maintainers/triagers) can already assign issues with GitHub UI. And I don't see regular users wanting to assign to someone else. And note that GitHub limits assignees to people who commented on the issue. That's the "fun" part, creating the comment with the take is what allows the action to assign to the commenter.

Reply:
k